src/sentry/static/sentry/app/utils/dates.tsx

```diff
@@ -212,3 +212,8 @@ export function parsePeriodToHours(str: string): number {
       return -1;
   }
 }
+
+export const use24Hours = (): boolean => {
+  const user = ConfigStore.get('user');
+  return user && user.options && user.options.clock24Hours;
```

Comment:
You can use the null traversla operator here.

```ts
export const use24Hours = (): boolean => ConfigStore.get('user')?.options?.clock24Hours;
```

Reply:
Yesssss 1-liner!